main.py

```diff
@@ -18,17 +17,17 @@
 
 def printlogo():
     pc.printout("________         .__        __                               \n", pc.YELLOW)
-    pc.printout("\_____  \   _____|__| _____/  |_  ________________    _____  \n", pc.YELLOW)
-    pc.printout(" /   |   \ /  ___/  |/    \   __\/ ___\_  __ \__  \  /     \ \n", pc.YELLOW)
-    pc.printout("/    |    \\\___ \|  |   |  \  | / /_/  >  | \// __ \|  Y Y  \\\n", pc.YELLOW)
-    pc.printout("\_______  /____  >__|___|  /__| \___  /|__|  (____  /__|_|  /\n", pc.YELLOW)
-    pc.printout("        \/     \/        \/    /_____/            \/      \/ \n", pc.YELLOW)
+    pc.printout("\\_____  \\   _____|__| _____/  |_  ________________    _____  \n", pc.YELLOW)
```

Comment:
```suggestion
    pc.printout(r"\_____  \   _____|__| _____/  |_  ________________    _____  \n", pc.YELLOW)
```
Raw strings in Python allow you to drop the double backslashes.